src/solvers/refinement/refine_arithmetic.cpp

```diff
@@ -458,7 +458,7 @@ void bv_refinementt::check_UNSAT(approximationt &a)
 }
 
 /// check if an under-approximation is part of the conflict
-bool bv_refinementt::is_in_conflict(approximationt &a)
+bool bv_refinementt::conflicts_with(approximationt &a)
 {
   for(std::size_t i=0; i<a.under_assumptions.size(); i++)
```

Comment:
you can used ranged for

Reply:
Indeed I could. But this is just a function rename to make clang compile without warnings.